packages/xstate-react/test/useMachine.test.tsx

```diff
@@ -163,34 +163,33 @@ describeEachReactMode('useMachine (%s)', ({ suiteKey, render }) => {
     render(<Test />);
   });
 
-  // TODO: make new test with input (not related to options.context)
-  // it('should merge machine context with options.context', () => {
-  //   const testMachine = createMachine<{ foo: string; test: boolean }>({
-  //     context: ({ input }) => ({
-  //       foo: 'bar',
-  //       test: input.test ?? false
-  //     }),
-  //     initial: 'idle',
-  //     states: {
-  //       idle: {}
-  //     }
-  //   });
-
-  //   const Test = () => {
-  //     const [state] = useMachine(testMachine, {
-  //       input: { test: true }
-  //     });
-
-  //     expect(state.context).toEqual({
-  //       foo: 'bar',
-  //       test: true
-  //     });
-
-  //     return null;
-  //   };
-
-  //   render(<Test />);
-  // });
+  it('should merge machine context with options.context', () => {
+    const testMachine = createMachine<{ foo: string; test: boolean }>({
+      context: ({ input }) => ({
+        foo: 'bar',
+        test: input.test ?? false
+      }),
+      initial: 'idle',
+      states: {
+        idle: {}
+      }
+    });
+
+    const Test = () => {
+      const [state] = useMachine(testMachine, {
+        input: { test: true }
```

Comment:
we should add the same test to all other integration packages

Reply:
https://linear.app/statelyai/issue/STA-3445